gateway/main/settings.py

```diff
@@ -95,15 +95,33 @@
 LOGGING = {
     "version": 1,
     "disable_existing_loggers": False,
+    "formatters": {
+        "simple": {
+            "format": "%(levelname)s %(asctime)s %(filename)s:%(lineno)s : %(message)s"
+        },
+    },
     "handlers": {
         "console": {
             "class": "logging.StreamHandler",
+            "formatter": "simple",
         },
     },
     "root": {
         "handlers": ["console"],
         "level": "DEBUG",
     },
+    "loggers": {
```

Comment:
@IceKhan13 what do you think about include an additional logger called `scheduler`? I see interesting to differentiate between `gateway` and `scheduler`.